csa-runner-app/src/test/resources/application.yml

```diff
@@ -53,9 +53,13 @@ rao-runner-server:
     # must therefore stay in queue until a rao-runner is free, or until the time-out is reached
     queue-name: swe-csa-rao-request-queue
 
+
 dichotomy-parameters:
-  index:
-    precision: 10
-    max-iterations-by-border: 10
+  precision: 10
```

Comment:
I think that if we need different max iterations on the 2 borders, we should also have different precisions on the 2 borders?